test/Microsoft.AspNetCore.SignalR.Client.Tests/HttpConnectionTests.Transport.cs

```diff
@@ -66,6 +66,46 @@ public class Transport
                 Assert.True(requestsExecuted);
             }
 
+            [Theory]
+            [InlineData(HttpTransportType.LongPolling)]
+            [InlineData(HttpTransportType.ServerSentEvents)]
```

Comment:
Can you check how WebSockets behaves (as we discussed in person)? I think it may need this header too to avoid the 302.

Reply:
👍 
We're going to also set the header on httpOptions so that WebSockets gets the header set too

Reply:
I wrote an endpoint that echos back a HTTP header for testing the user agent. Checkout WebSocketsTransportSendsUserAgent. You should be able to test web sockets passing the header with the same logic.